tools/server-all/pom.xml

```diff
@@ -240,14 +240,13 @@
                   <tasks>
                     <antversion property="antversion" />
                     <echo>Ant Version: ${antversion}</echo>
-
                     <property name="appserver.id" value="tomcat8" />
                     <property name="apiman.tomcat.download.url"
-                      value="http://apache.cs.utah.edu/tomcat/tomcat-8/v8.0.33/bin/apache-tomcat-8.0.33.zip" />
+                      value="http://apache.cs.utah.edu/tomcat/tomcat-8/v8.5.51/bin/apache-tomcat-8.5.51.zip" />
```

Comment:
I am not sure what these `server-tools` exactly do, but have you tried if we are getting problems if we would upgrade to tomcat 9 or even 10?

Reply:
I didn't try to upgrade the version of tomcat, I only fixed it to optimize and make easer my tests, but I'll try it in a few days.